sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/HiveMetastoreLazyInitializationSuite.scala

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
This test is in hive-thriftserver is because tests in the hive project use a shared singleton HiveContext and I cannot create a new one in the same project.